app/src/main/java/wordtextcounter/details/main/store/entities/Report.kt

```diff
@@ -0,0 +1,15 @@
+package wordtextcounter.details.main.store.entities
+
+import android.arch.persistence.room.ColumnInfo
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+
+@Entity(tableName = "Details")
+data class Report(
+    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "id") var id: Int?,
+    @ColumnInfo(name = "name") var name: String?,
+    @ColumnInfo(name = "data_text") var dateText: String?,
+    @ColumnInfo(name = "word") var words: String?,
```

Comment:
Word char etc can be stored as an to Int

Reply:
Makes sense. Can do.